action.yml

```
@@ -42,6 +42,8 @@ outputs:
     description: 'Generated Docker labels'
   bake-file:
     description: 'Bake definiton file'
+  json:
+    description: 'A JSON object composed of the generated tags and labels'
```

Better to keep the same desc of the README: https://github.com/docker/metadata-action#outputs

```suggestion
    description: 'JSON output of tags and labels'
```